### DISCUSSION TITLE
How to update MXCSR automatically?

### DISCUSSION THREAD

**Discussion by ALanovaya** [unicorn-engine/unicorn#1981] [Q&A]
I am emulating code on x86 that contains instructions working with floating point numbers, but for some reason during emulation the mxcsr register is not updated in any way, although I specifically emulated examples (for example, division by zero), where mxcsr was definitely supposed to change, but this did not happen
Why is that ?

**Comment by wtdcode** [comment]
I created an issue for you. I didn't actively check discussion so please raise an issue next time.
